tests/modules/nf-core/ashlar/main.nf

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { ASHLAR } from '../../../../modules/nf-core/ashlar/main.nf'
+include { ASHLAR as ASHLAR_TILE } from '../../../../modules/nf-core/ashlar/main.nf'
+
+// we zero out the UUID of output tiff images with ZERO_UUID so we get a consistent md5sum
+include { ZERO_UUID } from './zero_uuid.nf'
```

Comment:
The output from ASHLAR contains a UUID that is different between runs.  To get the same md5 sum for each file we use zero_uuid.nf to set all values of the UUID to zero.  The input to zero_uuid.nf is an offset to the UUID location in the file.
Is this okay or is there a better way of handling it?

Reply:
Where is the UUID? Embedded into the file somehow?
Does that mean that the tool produces a different output every time it is ran? Because if so, that might be an issue in general for pipeline reproducibility.

Reply:
Yes, it creates a UUID in the header of the image file output.  That is the only thing that is different between runs because date and time are used in creating the UUID.  I realize we could have just checked for the existence of the file, but since this is the only element of the file that differs, we would like to make sure the actual image contents of the file have not changed between runs and this solution gives us that.

Reply:
The OME-XML schema used to store metadata in the TIFF header has a UUID field and Ashlar fills that field with a random UUID. We had considered making a change to Ashlar itself to support passing an explicit UUID on the command line and/or disabling UUID generation completely (the XML schema does make the UUID optional) but concluded that just patching the nf-core module tests was more expedient. If and when Ashlar makes that change, we'll remove the zero_uuid code here.

Reply:
Was there any progress on updating the tool on the command line so this isn't necessary?